cmd/xl-storage.go

```diff
@@ -558,11 +559,21 @@ func (s *xlStorage) SetDiskID(id string) {
 }
 
 func (s *xlStorage) MakeVolBulk(ctx context.Context, volumes ...string) (err error) {
-	for _, volume := range volumes {
-		if err = s.MakeVol(ctx, volume); err != nil {
-			if osIsPermission(err) {
-				return errVolumeAccessDenied
+	g := errgroup.WithNErrs(len(volumes))
+	for index := range volumes {
+		index := index
+		g.Go(func() error {
```

Comment:
Maybe a lot of goroutines will be created if we have a lot of buckets ?

Reply:
correct are you saying we should do 100 at a time?

Reply:
simplified it @vadmeste PTAL